src/main/java/org/cryptacular/CiphertextHeader.java

```diff
@@ -67,12 +75,17 @@ public CiphertextHeader(final byte[] nonce)
    */
   public CiphertextHeader(final byte[] nonce, final String keyName)
   {
-    this.nonce = nonce;
-    this.length = 8 + nonce.length;
+    if (nonce.length > 255) {
+      throw new IllegalArgumentException("Nonce exceeds size limit in bytes (255)");
```

Comment:
Can use the constant in the previous two lines.

Reply:
Fixed.